templates/service-control-policies.yaml

```diff
@@ -19,11 +19,15 @@ Parameters:
 Conditions:
   IncludeSecurityHub: !Equals [ !Ref IncludeSecurityHub, true ]
   IncludeBackup: !Equals [ !Ref IncludeBackup, true ]
+  SCPwithSecurityAndBackup: !And
+    - !Equals [ !Ref IncludeSecurityHub, true ]
+    - !Equals [ !Ref IncludeBackup, true ]
 
 Resources:
 
   ServiceControlPolicy:
     Type: AWS::CloudFormation::CustomResource
+    Condition: SCPwithSecurityAndBackup
```

Comment:
Maybe rename the condition to a more generic name like `CreateServiceControlPolicy`